packages/sanity/src/core/create/start-in-create/CreateLinkingDialog.tsx

```diff
@@ -25,7 +25,7 @@ export function CreateLinkingDialog() {
   }, [])
 
   return (
-    <Dialog id={id} width={0} header={t('linking-in-progress-dialog.header')}>
+    <Dialog header={t('linking-in-progress-dialog.header')} id={id} width={0}>
```

Comment:
😄

Reply:
Apols for the noise, this is a terrible habit of mine (ordering props alphabetically) after the fact